### PR TITLE
Mention the price for the Quick Start session

### PR DESCRIPTION
**Pull request by claudiucelfilip**
#### Changes proposed in this Pull Request

This PR changes **Get personalized help** copy to include the price of the Quick Start session.
See pbBQWj-eI-p2 for more details.

Screenshot on 2020-07-29 at 16-27-15

#### Testing instructions

* Purchase a new Business plan or upgrade.
* On the Thank You pages which appears after Checkout
* The section **Get personalised help** should include the price of the Quick Start session.
Screenshot on 2020-07-29 at 16-23-16

### CODE REVIEW

**Comment by travisw** [comment]
Should this also be applying the change for [your last PR for the thank you page](https://github.com/Automattic/wp-calypso/pull/44516) that hides this block for non-English users?
Currently I'm see the block when using a non-English locale, which is resulting both non-English and English on the page.

Screen Shot 2020-07-31 at 6 13 20 PM

**Comment by claudiucelfilip** [comment]
> Should this also be applying the change for [your last PR for the thank you page](https://github.com/Automattic/wp-calypso/pull/44516) that hides this block for non-English users?

I applied the change from [that PR](https://github.com/Automattic/wp-calypso/pull/44516).
For the text with the price, I set a similar check but with a fallback to the translation without the price. Since the 2 versions are very similar, I though it would be a shame to hide everything just because of the pricing.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4179738

Thank you @claudiucelfilip for including a screenshot in the description! This is really helpful for our translators.

File: client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx [hunk 1]
```diff
@@ -121,5 +152,10 @@ BusinessPlanDetails.propTypes = {
 	selectedFeature: PropTypes.object,
 	sitePlans: PropTypes.object.isRequired,
 };
-
-export default BusinessPlanDetails;
+export default connect( ( state ) => {
+	const productsList = getProductsList( state );
+	return {
+		hasProductsList: Object.keys( productsList ).length > 0,
+		productDisplayCost: getProductDisplayCost( state, 'concierge-session' ),
```
[the remark this answers is not in the document]
Comment (claudiucelfilip) [reply client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx]: Good point. Changed it to `conciergeSessionDisplayCost`.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4179738).

**Pull request merged**